src/govuk/components/pagination/template.njk

```diff
@@ -0,0 +1,62 @@
+{% set blockLevel = not params.items and (params.next or params.previous) %}
+
+<nav class="govuk-pagination {{- ' ' + params.classes if params.classes }} {{- ' govuk-pagination--block' if blockLevel }}" role="navigation" aria-label="{{ params.landmarkLabel | default("results") }} {%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}" {%- endfor -%}">
+  {%- if params.previous and params.previous.href -%}
+    <div class="govuk-pagination__prev">
+      <a class="govuk-link govuk-pagination__link" href="{{ params.previous.href }}" rel="prev" {%- for attribute, value in params.previous.attributes %} {{ attribute }}="{{ value }}" {%- endfor -%}>
+        <svg class="govuk-pagination__icon govuk-pagination__icon--prev" xmlns="http://www.w3.org/2000/svg" height="13" width="15" aria-hidden="true" focusable="false" viewBox="0 0 15 13">
+          <path d="m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z"></path>
+        </svg>
+        <span class="govuk-pagination__link-title {{- ' govuk-pagination__link-title--decorated' if blockLevel and not params.previous.labelText }}">
+          {{- params.previous.text | default("Previous") -}}
+        </span>
+        {%- if params.previous.labelText and blockLevel -%}
+          <span class="govuk-visually-hidden">:</span>
+          <span class="govuk-pagination__link-label">{{ params.previous.labelText }}</span>
+        {%- endif -%}
+      </a>
+    </div>
+  {% endif %}
+
+  {%- if params.items -%}
+    <ul class="govuk-pagination__list">
+      {%- for item in params.items -%}
+        {%- if item.ellipsis -%}
+          <li class="govuk-pagination__item govuk-pagination__item--ellipses">&ctdot;</li>
+        {%- elseif item.number -%}
+          <li class="govuk-pagination__item {{- ' govuk-pagination__item--current' if item.current }}">
+            <a class="govuk-link govuk-pagination__link" href="{{ item.href }}" aria-label="{{ item.visuallyHiddenText | default("Page " + item.number) }}" {%- if item.current %} aria-current="page" {%- endif -%} {%- for attribute, value in item.attributes %} {{ attribute }}="{{ value }}" {%- endfor -%}>
```

Comment:
Does this mean that the `visuallyHiddenText` for items should include the actual page number (eg `Result 1`) rather than just the word to replace "Page"? If so, it might be worth making this clear in the macro guidance?